### DISCUSSION TITLE
I cant build project

### DISCUSSION THREAD

**Discussion by axodevjs** [OneKeyHQ/app-monorepo#4474] [Q&A]
The error is:
Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
> Configure project :react-native-ble-manager
Build file 'C:\Users\samoi\Documents\projects\test\app-monorepo\node_modules\react-native-ble-manager\android\build.gradle': line 42
The RepositoryHandler.jcenter() method has been deprecated. This is scheduled to be removed in Gradle 9.0. JFrog announced JCenter's sunset in February 2021. Use mavenCentral() instead. Consult the upgrading guide for further information: https://docs.gradle.org/8.3/userguide/upgrading_version_6.html#jcenter_deprecation
        at build_c7s73h0jqptumm7rpwloe2owh$_run_closure2.doCall(C:\Users\samoi\Documents\projects\test\app-monorepo\node_modules\react-native-ble-manager\android\build.gradle:42)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
        at build_c7s73h0jqptumm7rpwloe2owh.run(C:\Users\samoi\Documents\projects\test\app-monorepo\node_modules\react-native-ble-manager\android\build.gradle:39)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)

./gradlew -v

------------------------------------------------------------
Gradle 8.3
------------------------------------------------------------

Build time:   2023-08-17 07:06:47 UTC
Revision:     8afbf24b469158b714b36e84c6f4d4976c86fcd5

Kotlin:       1.9.0
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          20.0.2 (Oracle Corporation 20.0.2+9-78)
OS:           Windows 11 10.0 amd64

**Comment by huhuanming** [comment]
Thank you very much for your feedback. Can you try again now to see if it can be rebuilt?
